app/styles/_core.less

```diff
@@ -1099,6 +1101,7 @@ status-icon {
   // so that status-icons have a fixed width (like .fa-fw, but better sized)
   // this ensures when the icons change, the text that follows doesn't shift
   .fa, .pficon {
+    font-size: 14px;
```

Comment:
I think we may want to scope this change to only the unexpanded rows as I'm noticing the status-icons seem a little wonky in other contexts now (like the expanded row where builds appear).